src/main/java/mate/academy/util/HashUtil.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+public class HashUtil {

```

Comment:
check common mistakes, we should add **private** default constructor to the `HashUtil`, since it's a util class with static-only methods